### PR TITLE
add helm chart

### PR DESCRIPTION
**Pull request by angelnu**
### Description

Create Helm chart for RasberryMatic to be installed into K8S

### Related Issue

#1090

### Types of changes

- [ ] Docs change / refactoring / dependency upgrade
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### Alternate Designs

Manually installing OCI into K8S

### Possible Drawbacks

New branch is required. Use the following commands:

1. `git checkout --orphan gh-pages `
2. `git rm -rf .`
3. `git commit -m "Initial commit" --allow-empty`
4. `git push --set-upstream origin gh-pages`

### Verification Process

After each succesfull build of master a matching Helm chart will be created.

For the first time please manually trigger using the `Helm chart - workflow dispatch` Workflow.

### Release Notes

I will add the instructions to the wiki once the first chart is created. Test with my fork worked fine.

Basically instuctions are:

1. helm repo add raspberrymatic https://angelnu.github.io/RaspberryMatic/
2. create /tmp/ccu  folder in K8S server (if no default claim class is available)
3. helm install --set persistence.hostPath=/tmp/ccu my-ccu raspberrymatic/raspberrymatic

### Contributing checklist

- [x] My code follows the code style of this project.
- [x] I have read the **CONTRIBUTING** and **LICENSE** document.
- [x] I fully agree to distribute my changes under Apache 2.0 license.

### CODE REVIEW

**Comment by angelnu** [comment]
I cherry picked the fix for the deploy test so this PR can be tested overnight
